enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/ShiroAuthManager.java

```diff
@@ -216,6 +213,18 @@ public boolean deleteUser( String username ) throws IOException
         return realm.deleteUser( username );
     }
 
+    void suspendUser( String username ) throws IOException, ConcurrentModificationException
+    {
+        assertAuthEnabled();
+        realm.suspendUser( username );
+    }
+
+    void activateUser( String username ) throws IOException, ConcurrentModificationException
```

Comment:
We don't want to expose the `ConcurrentModificationException` (see comments in FileUserRealm).